geonode/harvesting/harvesters/base.py

```diff
@@ -173,6 +173,16 @@ def update_geonode_resource(
                     f"the remote resource {harvested_info.resource_descriptor.uuid!r}")
             geonode_resource = resource_manager.update(
                 str(harvested_info.resource_descriptor.uuid), vals=defaults)
+
+        # update keywords and regions
+        keywords = list(
+            harvested_info.resource_descriptor.identification.other_keywords
+        ) + geonode_resource.keyword_list()
+        keywords.append(harvester.name.lower())
```

Comment:
As stated in the issue description, please be sure to remove the `harvester` string if it is contained in the name

Reply:
updatd @ricardogsilva